src/common/config_cacher.h

```diff
@@ -31,28 +31,28 @@
 template <typename ValueT>
 class md_config_cacher_t : public md_config_obs_t {
   ConfigProxy& conf;
-  const char* keys[2];
+  const std::string option_name;
```

Comment:
Yeah, this is so cold part of the code that employing `std::string` seems purely beneficial.